packages/app/src/components/common/UpdateBanner/UpdateBanner.tsx

```diff
@@ -0,0 +1,57 @@
+import { Alert, Box } from '@mui/material'
+import { useLatestVersion } from '../../../api/version.api'
+import { useLocalStorage } from '../../../hooks/useBrowserStorage'
+import { CodeSnippit } from '../CodeSnippit/CodeSnippit'
+
+const wasOverTwenyFourHoursAgo = (dismissedAt: number) => {
+  const twentyFourHours = 1_000 * 60 * 60 * 24
+  return new Date().getTime() - dismissedAt >= twentyFourHours
+}
+
+export const UpdateBanner = () => {
+  const localVersion = __APP_VERSION__
+  const [dismissedInfo, setDismissedInfo] = useLocalStorage({
+    key: 'update-banner-dismissed',
```

Comment:
If they dismiss an update one time, will they never be notified after that of new updates?

Reply:
With how I have it set up it will bug them every 24 hours.

Reply:
haha, cool